content/docs/reference/pkg/eks/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "eks"
+title_tag: "Module eks | Package eks"
+meta_desc: "Pulumi Amazon Web Services (AWS) EKS Components."
+menu:
+    reference:
+        parent: API Reference
+---
+
+<!-- WARNING: this file was generated by Pulumi Docs Generator. -->
+<!-- Do not edit by hand unless you're certain you know what you are doing! -->
+
+Pulumi Amazon Web Services (AWS) EKS Components.
+
+<h2 id="resources">Resources</h2>
+<ul class="api">
+    <li><a href="cluster" title="Cluster"><span class="symbol resource"></span>Cluster</a></li>
+    <li><a href="clustercreationroleprovider" title="ClusterCreationRoleProvider"><span class="symbol resource"></span>ClusterCreationRoleProvider</a></li>
+    <li><a href="managednodegroup" title="ManagedNodeGroup"><span class="symbol resource"></span>ManagedNodeGroup</a></li>
+    <li><a href="nodegroup" title="NodeGroup"><span class="symbol resource"></span>NodeGroup</a></li>
+    <li><a href="nodegroupsecuritygroup" title="NodeGroupSecurityGroup"><span class="symbol resource"></span>NodeGroupSecurityGroup</a></li>
+    <li><a href="provider" title="Provider"><span class="symbol resource"></span>Provider</a></li>
```

Comment:
In EKS, I'm currently deleting the generated `Provider` resource as I didn't think it'd be useful for this library. But forgot that it would show up here in the docs. I suppose I'll just go ahead and bring this back in EKS.